Makefile

```diff
@@ -130,14 +133,18 @@ ami-check: generate-ami ## Check whether the AMIs have been updated and fail if
 
 .PHONY: generate-kubernetes-types
 generate-kubernetes-types: ## Generate Kubernetes API helpers
-	@build/vendor/k8s.io/code-generator/generate-groups.sh deepcopy,defaulter _ github.com/weaveworks/eksctl/pkg/apis eksctl.io:v1alpha5
+	@go mod download k8s.io/code-generator # make sure the code-generator is present
+	@echo "/*\n$$(cat LICENSE)*/\n" > codegenheader.txt
+	@env GOPATH="$$(go env GOPATH)" bash "$$(go env GOPATH)/pkg/mod/k8s.io/code-generator@v0.0.0-20190612205613-18da4a14b22b/generate-groups.sh" \
```

Comment:
How is `go env GOPATH` different from simple variable lookup?

Reply:
`go env GOPATH` always provides a value, even if you don't declare `GOPATH` (which you don't need to with go >= 1.12)